site/src/components/pages/dashboard/TopNavigationTabs.tsx

```diff
@@ -0,0 +1,54 @@
+import { VoidFunctionComponent, MouseEvent } from "react";
+import { Box, Container, Tabs, Tab } from "@mui/material";
+import { useRouter } from "next/router";
+
+import { dashboardPages } from "./utils";
+
+const a11yProps = (index: number) => ({
+  id: `simple-tab-${index}`,
+  "aria-controls": `simple-tabpanel-${index}`,
+});
+
+export const TopNavigationTabs: VoidFunctionComponent = () => {
+  const router = useRouter();
+
+  return (
+    <Box
+      sx={{
+        borderBottom: 1,
+        borderColor: ({ palette }) => palette.gray[20],
+        borderBottomStyle: "solid",
+        marginTop: {
+          xs: 2,
+          md: 0,
+        },
+      }}
+    >
+      <Container>
+        <Tabs
+          value={
+            dashboardPages.find((dashboardPage) =>
```

Comment:
This is a heuristic way to determine the current page, so it doesn't get affected by hashes or query parameters. This component in itself could eventually be created into a generic reusable component.